app/src/main/java/org/mozilla/fenix/compose/CollectionsPlaceholder.kt

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.ext.dashedBorder
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+/**
+ * [CollectionsPlaceholder] for displaying a message detailing the collections feature and
+ * allowing users to easily start creating their first.
+ *
+ * @param showAddToCollectionButton Whether or not the "Add to Collection" button should be shown.
+ * @param onAddTabsToCollectionButtonClick Invoked when the user clicks on the "Add Tabs to Collection" button.
+ * @param onRemoveMessageClick Invoked when the user clicks on the close button to remove the Collections placeholder.
+ */
+@Composable
+fun CollectionsPlaceholder(
+    showAddToCollectionButton: Boolean,
+    onAddTabsToCollectionButtonClick: () -> Unit,
+    onRemoveMessageClick: () -> Unit,
+) {
+    Box(
+        modifier = Modifier
+            .semantics(mergeDescendants = true) {}
+            .dashedBorder(
+                color = FirefoxTheme.colors.borderPrimary,
+                cornerRadius = 8.dp,
+                dashHeight = 2.dp,
+                dashWidth = 4.dp
+            )
+            .padding(
+                start = 16.dp,
+                bottom = 16.dp
+            )
```

Comment:
The padding ain't even on horizontal and vertical axis. You can see the content is shifted to the right. I believe you can remove the entire `padding()` declaration here since the 16dp padding is already handled in the Column.

<img width="465" alt="Screen Shot 2022-04-19 at 4 35 29 PM" src="https://user-images.githubusercontent.com/1190888/164091267-e2bb72ef-46cc-469f-8929-7c200a72d699.png">